app/src/main/java/org/hwyl/sexytopo/model/graph/Coord2D.java

```diff
@@ -27,10 +27,30 @@ public Coord2D minus(Coord2D coord2D) {
         return new Coord2D(this.x - coord2D.x, this.y - coord2D.y);
     }
 
+    public Coord2D multiply(Coord2D coord2D) {
+        return new Coord2D(this.x * coord2D.x, this.y * coord2D.y);
+    }
+
+    public Coord2D div(double divisor) {
```

Comment:
Do we need a `div` function? Could just `scale()` by `-mag`?

Reply:
Errrr that's not how maths works

Reply:
Oh yeah. You know what I meant :stuck_out_tongue: